docs/source/guides/cache_system.mdx

```diff
@@ -12,24 +12,26 @@ specific language governing permissions and limitations under the License.
 
 # Neuron Model Cache
 
-The Neuron Model Cache is a remote cache for compiled Neuron models in the `neff` format.
-It is integrated into the `NeuronTrainer` and `NeuronModelForCausalLM` classes to enable loading pretrained models from the cache instead of compiling them locally.
-
-**Note: it is not available for models exported using any other NeuronModelXX classes, that use a different export mechanism.**
+The Neuron Model Cache is a remote cache for compiled Neuron models in the `neff` or `.neuron`(TorchScript) format.
+It is integrated into the `NeuronTrainer` and `NeuronModelForXXX` classes to enable loading pretrained models from the cache instead of compiling them locally.
 
 The Neuron Model Cache is hosted on the [Hugging Face Hub](https://huggingface.co/aws-neuron/optimum-neuron-cache) and includes compiled files for all popular and supported `optimum-neuron` pre-trained models.
 
-Before training a Transformers or Diffusion model or loading a NeuronModelForCausalLM on Neuron platforms, it needs to be exported to neuron format
+Before training a Transformers or Diffusion model or loading a NeuronModelForXXX on Neuron platforms, it needs to be exported to neuron format
 with [`torch-neuronx`](https://github.com/aws-neuron/aws-neuron-samples/tree/master/torch-neuronx).
 
-When exporting a model, [`torch-neuronx`](https://github.com/aws-neuron/aws-neuron-samples/tree/master/torch-neuronx) will:
+When exporting a model, [`torch-neuronx`](https://github.com/aws-neuron/aws-neuron-samples/tree/master/torch-neuronx) will either:
+
+- 1st approach: convert it to a set of [XLA](https://github.com/pytorch/xla/) subgraphs, and compile each subgraph with the neuronx compiler into a Neuron Executable File Format (NEFF) binary file(eg. training and inference of llm). 
+
+Or
+
+- 2nd approach: recorde one graph, compile it with the neuronx-cc compiler to a NEFF binary file, and serialize as a TorchScript module(eg. inference of encoders, stable diffusion and seq2seq models).
```

Comment:
```suggestion
- 2nd approach: recorde one graph via tracing (TorchScript), compile it with the neuronx compiler to a NEFF binary file, and serialize as a TorchScript module(eg. inference of encoders, stable diffusion and seq2seq models).
```
Between the two bullet points you use "neuronx compiler" and "neuronx-cc compiler". I think the former is more accurate.